base_location/views/res_city.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="view_city_tree_inherit" model="ir.ui.view">
+        <field name="model">res.city</field>
+        <field name="inherit_id" ref="base_address_city.view_city_tree"/>
+        <field name="arch" type="xml">
+            <tree position="attributes">
+                <attribute name="editable"/>
+            </tree>
+            <field name="zipcode" position="replace">
```

Comment:
Please don't replace, but hide the element, or we expose to a possible error if other module takes this element as reference.